BUILD.gn

```diff
@@ -106,6 +106,7 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
     if (chip_enable_python_modules) {
       deps = [
         "${chip_root}/scripts:matter_yamltests_distribution.wheel",
+        "${chip_root}/scripts/tests/py:metadata_parser.wheel",
```

Comment:
question, possibly for @andreilitvin - why do we have a single script that's a separate wheel file?

Reply:
@andy31415

Reply:
I think the intent should be to pull in all testing support/infra files separately, so I imagine we want eventually something that contains matter_testing_support and global_attribute_ids and conformance_support and such (and we really should remove the `support` smurf suffix from them when we move).

For this I proposed to have a different name for things as a result: no more `py` in the module name (but maybe it should be in a folder name since pigweed seems to do it often (like https://github.com/google/pigweed/tree/main/pw_bloat or https://github.com/google/pigweed/tree/main/pw_watch) ... then again they prepare for different folder structures.

Overall I would like to be able to `import matter_testing_infrastructure.global_attribute_ids` instead of the current `import global_attribute_ids`.

Reply:
So is the plan to move the python testing framework files into the scripts directory? 

I'd much prefer to leave them where they are (maybe in a sub directory, but at least close). Developers know where to find these files. If you want the metadata parser to be part of the testing wheel, it should be moved to be with the testing stuff and not the other way around.

But honestly, I don't think this really feels like the same thing - the metadata parser is just a CI script, whereas the testing stuff feels like a cohesive module.

Reply:
we can place them in testing as well. They are indeed separated enough purpose-wise, however it also seems we want to minimize the number of wheels overall. So maybe separate packages in the same wheel?

Reply:
marking resolved: the root path was now moved.